docs/advanced_usages/scaling.md

```diff
@@ -0,0 +1,635 @@
+export ANTA_CONNECT_TIMEOUT=30.0
+
+# Maximum duration to wait for a chunk of data to be received (for example, a chunk of the response body)
+export ANTA_READ_TIMEOUT=30.0
+
+# Maximum duration to wait for a chunk of data to be sent (for example, a chunk of the request body)
+export ANTA_WRITE_TIMEOUT=30.0
+
+# Maximum duration to wait for acquiring a connection from the connection pool
+export ANTA_POOL_TIMEOUT=30.0
+```
+
+If not set, these values default to the global `timeout` of ANTA, which is `30.0` seconds by default. The global timeout can also be set via the `--timeout` command-line option. See [HTTPX documentation](https://www.python-httpx.org/advanced/timeouts/) for more details.
+
+### Performance Tuning
+
+For optimal single-process performance, consider the following tuning parameters:
+
+1. Adjust concurrency based on test count and system resources:
+
+    ```bash
+    # For 250 devices with 100 tests each (25,000 total tests)
+    export ANTA_MAX_CONCURRENCY=25000
+    ```
+
+    Even though the default value is `10000`, it may not be optimal for your system. You can increase or decrease this value gradually to find the best setting for your environment.
+
+2. Optimize connection pooling:
+
+    ```bash
+    export ANTA_MAX_CONNECTIONS=10
+    export ANTA_MAX_KEEPALIVE_CONNECTIONS=5
+    ```
+
+    These values are a good starting point for large fabrics. Adjust them based on your fabric size and performance testing.
+
+3. Increase timeouts for stability:
+
+    ```bash
+    export ANTA_TIMEOUT=3600.0
+    ```
+
+    This is **very** important for large fabrics. Increase the global timeout to prevent test failures due to timeouts. You shouldn't need to adjust the other timeout values unless you have specific requirements.
+
+## 🔀 Multi-Process Scaling
+
+When single-process optimization isn't enough, you can scale ANTA horizontally by running multiple instances in parallel. This approach is particularly useful for large fabrics with 500+ devices. This section will provide a few examples of how to achieve this using the common tool [`GNU Parallel`](https://www.gnu.org/software/parallel/).
+
+By leveraging `GNU Parallel` and proper test organization, we can significantly reduce execution time and improve overall performance while maintaining accurate test results.
+
+### Prerequisites
+
+Before implementing multi-process scaling:
+
+- 🏗️ Understand your fabric topology
+- 🎯 Identify natural test boundaries (e.g., devices, PODs, roles, etc.)
+- 📋 Plan your test catalog(s) accordingly
+- 💻 Understand basic Linux command-line operations
+
+### Installation Requirements
+
+!!! info "System Requirements"
+    The following procedures and examples were tested on **Ubuntu 22.04**, using `GNU Parallel` version `20241022` and `yq` version `v4.44.3`. Please refer the tool's documentation for specific installation instructions.
+
+#### Installing Required Tools
```

Comment:
title is not needed as there is no other sub title (or maybe should be called `Ubuntu`